.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: "Getting Help"
+    about: "Links to documentation and community support"
+    url: "https://cloudinit.readthedocs.io/en/latest/#project-and-community"
```

Comment:
This will need to be updated as soon as we complete the migration to rtd.com (not much to be done now, but just as a reminder)